### DISCUSSION TITLE
how to access settings in /interface/ethernet/switch?

### DISCUSSION THREAD

**Discussion by cbc02009** [terraform-routeros/terraform-provider-routeros#282] [Q&A] [answered]
I'm setting up some switches running routerOS. When I manually configure them I have to set
```
/interface ethernet switch
set 0 l3-hw-offloading=yes
```
How do I achieve similar with this terraform provider?

**Comment by vaerh** [comment]
Unfortunately, there is no resource at the provider at this time that can help you with this.
I'm a bit overloaded lately and can't find time to solve existing problems. For this reason there are two ways out: if you know Go and a little terraform SDK try to implement a resource and I am ready to review it and accept it in the main branch.
The other option, but obviously not fast, please open the issue in the repository.

**Comment by vaerh** [comment, marked as answer]
Solved.

The release is available on [GitHub release](https://github.com/terraform-routeros/terraform-provider-routeros/releases/tag/v1.21.0)

**Reply by cbc02009** [reply]
Thank you!
